src/OrchardCore.Modules/OrchardCore.Admin/AdminFilter.cs

```diff
@@ -21,13 +21,18 @@ public AdminFilter(IAuthorizationService authorizationService)
 
         public override async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
```

Comment:
Out of curiosity, do you know if this filter is called after or before the authorization filters? If it's called after, it won't probably work well with admin pages that use bearer as the authentication scheme (is that a thing, tho?)

Reply:
Admin controllers are very specific to orchard backend views. I am fine forcing users to create other controllers if they need to expose APIs.